Description: Add "Chromium" and "Ubuntu" to the user-agent text.

Index: dev.raring/src/webkit/user_agent/user_agent_util.cc
===================================================================
--- dev.raring.orig/src/webkit/user_agent/user_agent_util.cc	2013-04-23 06:20:37.856153269 -0500
+++ dev.raring/src/webkit/user_agent/user_agent_util.cc	2013-04-23 06:21:00.974499066 -0500
@@ -170,13 +170,21 @@
   // Derived from Safari's UA string.
   // This is done to expose our product name in a manner that is maximally
   // compatible with Safari, we hope!!
+
+  // We want to also add Chromium
+  std::string realproduct = product.c_str();
+  size_t pos;
+  if ((pos = realproduct.find("/")) != std::string::npos)
+      realproduct.replace(0, pos, "Chromium");
+
   std::string user_agent;
   base::StringAppendF(
       &user_agent,
-      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) %s Safari/%d.%d",
+      "Mozilla/5.0 (%s) AppleWebKit/%d.%d (KHTML, like Gecko) Ubuntu %s %s Safari/%d.%d",
       os_info.c_str(),
       WEBKIT_VERSION_MAJOR,
       WEBKIT_VERSION_MINOR,
+      realproduct.c_str(),
       product.c_str(),
       WEBKIT_VERSION_MAJOR,
       WEBKIT_VERSION_MINOR);
